osxcollector/osxcollector.py

```diff
@@ -879,6 +880,62 @@ def _version_string(self):
         """Log the current version of this program (osxcollector)"""
         Logger.log_dict({'osxcollector_version': __version__})
 
+    @_foreach_homedir
+    def _collect_ssh_info(self, homedir):
+        """Collect the values of authorized_keys and known_hosts in the default .ssh path for all users"""
+        file_path = pathjoin(homedir.path, '.ssh/authorized_keys')
+        if os.path.isfile(file_path):
+            with Logger.Extra('osxcollector_subsection', 'authorized_keys'):
+                self._log_authorized_keys(file_path)
+
+        file_path = pathjoin(homedir.path, '.ssh/known_hosts')
+        if os.path.isfile(file_path):
+            with Logger.Extra('osxcollector_subsection', 'known_hosts'):
+                self._log_known_hosts(file_path)
+
+    def _log_authorized_keys(self, filepath):
+        """Parse authorized_keys file
+
+        Args:
+            file_name: authorized_keys file
+            """
+        try:
+            with open(filepath, 'r') as fp:
+                for line in fp:
+                    if not line.startswith('#'):
+                        line.strip('/n')
+                        line = line.split()
+                        entry = {}
+                        entry['key_type'] = line[0]
```

Comment:
`authorized_keys` files can have more complicated formats ([see this specs](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.foto100/authkeyf.htm)) so better parsing than that would be required. That said, this tools is principally aimed at client machines, which (in most case) won't have SSH servers running on them, so you could just collect the `known_hosts`